tests/generation/test_modeling.py

```diff
@@ -145,3 +155,120 @@ def test_compare_with_and_without_past_key_values(self):
             f"With pkv latency: {with_pkv_timer.elapsed:.3f} ms, without pkv latency: {without_pkv_timer.elapsed:.3f} ms,"
             f" speedup: {without_pkv_timer.elapsed / with_pkv_timer.elapsed:.3f}",
         )
+
+
+class TSModelForSeq2SeqLMTest(unittest.TestCase):
+    SUPPORTED_ARCHITECTURES = ("t5",)
```

Comment:
thnaks for adding the tests ! Could we add tests for additionnal architectures ? (like bart, mbart, ..)